README.md

```diff
@@ -204,46 +191,48 @@ spec:
 ```
 
-For more information, see [Native X JVM Builds](#native-x-jvm-builds).
+For more information, see [Native X JVM builds](#native-x-jvm-builds).
+
+## Kogito Data Index Service deployment
+
 
-## Deploy Data Index Service
+The Kogito Operator can deploy the [Data Index Service](https://github.com/kiegroup/kogito-runtimes/wiki/Data-Index-Service) as a [Custom Resource](deploy/crds/app.kiegroup.org_v1alpha1_kogitodataindex_cr.yaml) (`KogitoDataIndex`).
 
-The Kogito Operator is able to deploy the [Data Index Service](https://github.com/kiegroup/kogito-runtimes/wiki/Data-Index-Service) as a [Custom Resource](deploy/crds/app.kiegroup.org_v1alpha1_kogitodataindex_cr.yaml) (`KogitoDataIndex`). 
+The Data Index Service depends on Kafka, so you must manually deploy an Apache Kafka Cluster, such as [Strimzi](https://strimzi.io/).
 
-Since Data Index Service depends on Kafka, it's necessary to manually deploy an Apache Kafka Cluster (we use [Strimzi](https://strimzi.io/)) in the same namespace.
- 
-Data Index also depends on Infinispan, but since 0.6.0 version, Infinispan Server is *automagically* deployed for you.
+The Data Index Service also depends on Infinispan, but starting in version 0.6.0 of the Kogito Operator, Infinispan Server is automatically deployed for you.
 
-| :information_source: It's planned for future releases that the Kogito Operator will deploy a Kafka cluster when deploying the Data Index Service. |
+
+| :information_source: In a future release, the Kogito Operator will deploy a Kafka cluster when deploying the Data Index Service. |
 | --- |
 
-### Deploy Infinispan
+### Deploying Infinispan
 
-If you plan to have the Data Index to connect to the Infinispan server deployed within the same namespace, Kogito Operator can handle this deployment for you.
+If you plan to use the Data Index Service to connect to an Infinispan Server instance deployed within the same namespace, the Kogito Operator can handle this deployment for you.
 
-When installing the Kogito Operator from OperatorHub, the Infinispan Operator will be available in the same namespace. Otherwise, if you don't have OperatorHub or OLM available in your cluster, you'll have to [subscribe to Infinispan Operator](https://infinispan.org/infinispan-operator/master/documentation/asciidoc/titles/operator.html#deploying_operator_manually) manually.
+When you install the Kogito Operator from OperatorHub, the Infinispan Operator is available in the same namespace. If you do not have access to OperatorHub or OLM in your cluster, you can [manually deploy the Infinispan Operator](https://infinispan.org/infinispan-operator/master/documentation/asciidoc/titles/operator.html#deploying_operator_manually).
 
-Having the Infinispan Operator deployed, just jump to [Install Data Index Service section](#install-data-index-service).
+After you deploy the Infinispan Operator, see [Kogito Data Index Service installation](#kogito-data-index-service-installation) for next steps.
```

Comment:
That link just skips the whole Strimzi deployment ... Is that on purpose ?

Reply:
@ricardozanini , since that's how it already was, please confirm.

Reply:
It's a mistake, good point @radtriste. This should be changed to "see Deploy Strimzi" for next steps. Good catch!

Reply:
Done.